src/main/app/case/CosApiClient.ts

```diff
@@ -139,13 +143,45 @@ export class CosApiClient {
           accessCode: 'Dummy accessCode',
         },
       });
+      const serviceauth = getServiceAuthToken();
+      logger.info(`Auth Token ****** ${user.accessToken} *******`);
+      logger.info(`service Token ****** ${serviceauth} *******`);
+      console.log(`Auth Token ****** ${user.accessToken} *******`);
+      console.log(`Auth Token ****** ${serviceauth} *******`);
 
       return { id: response.data.id, state: response.data.state, ...fromApiFormat(response.data) };
     } catch (err) {
       throw new Error('Case could not be updated.');
     }
   }
 
+  /* eslint-disable @typescript-eslint/no-explicit-any */
+  public async getAllHearingsForCitizenCase(user: UserDetails, caseId: string): Promise<any> {
```

Comment:
New method is not required. Change the existing method to pull hearing details from COS

Reply:
Yeah using the old mehtod now.